### ISSUE TITLE
Oxfordshire localisation: Paying offline / outside of transaction

### ISSUE THREAD

**Issue by suryanaga** [alphagov/verify-local-patterns#658]
![image](https://user-images.githubusercontent.com/4106955/30165310-803ddfc2-93d7-11e7-8ac8-dfde2801cdf9.png)

This page is not consistent with how Oxfordshire will be delivering their service. 

 1. The council will not be contacting users for payment, they will be emailing an invoice and a link to pay online. (This is because of technical constraints and will be changed in the future.)
 2. The council can not show the cost at this stage. The page should also explain that the cost needs to be calculated and will be included in the invoice that is emailed to them.

@lizziebruce Could you suggest revised content for this page?

**Comment by lizziebruce** [comment]
> **Payment summary**
> 
> When we’ve processed and approved your application, we’ll email you an invoice with a link to pay online. 
> 
> 
> You will be invoiced for:
> 
> [show all rows except cost]
> 
> We’ll calculate the cost of your permit. The invoice we send you will show the amount you need to pay.
> 
> NEXT

Question: will the emailed invoice also include option to pay by phone/post?

@AbiBarkerOxfordshire @sanjaypoyzer

**Comment by suryanaga** [comment]
No, it won't (I don't think). @abibarkeroxfordshire can confirm.

**Comment by suryanaga** [comment]
@artgov ^^ Above content should display on `/pre-payment` if `councilString == 'oxfordshire'`.

**Comment by AbiBarkerOxfordshire** [comment]
@sanjaypoyzer @lizziebruce Hi - thanks for this it all sounds good. Customers can pay by phone but we don't advertise it as an option - if a customer has an email address we email an invoice, if they don't provide one they we will call them (very few customers do not have an email address).

**Comment by suryanaga** [comment]
@abibarkeroxfordshire – Ok sure, there may be accessibility reasons why paying over the phone might be needed by some users, but that's something we should find out when we take a closer look at the payment solution in testing.
